core/stat/base/bucket_leap_array.go

```diff
@@ -37,6 +37,8 @@ func (bla *BucketLeapArray) NewEmptyBucket() interface{} {
 }
 
 func (bla *BucketLeapArray) ResetBucketTo(bw *BucketWrap, startTime uint64) *BucketWrap {
+	b := bla.currentBucketWithTime(util.CurrentTimeMillis())
+	b.reset()
```

Comment:
What's the purpose of the added line? Please see the below line`bw.Value.Store(NewMetricBucket())`.

Reply:
for reset the old bucket and create new one

Reply:
In my view, `ResetBucketTo ` only intend to reset `bw`.